### DISCUSSION TITLE
posts/2023/06/crisis-awareness/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#12] [General]
# posts/2023/06/crisis-awareness/

静静和专注。

https://guozheng.netlify.app/posts/2023/06/crisis-awareness/

**Comment by liechi** [comment]
按我自己的经验，学外语最有效的方法就是背诵和重复了，包括被单词和文章。背诵做到可以不想就能把内容脱口而出就好了，这是建立语感的途径之一。

>我加入了以万物有灵论、非人、物或人类学本体论为方向的小组。

这是一个组还是三个组？万物有灵，听起来很有意思，跟修仙似的。另外两个也很玄的样子。

**Reply by CyrusYip** [reply]
> 过去一直认为，背单词是一种比较低效的英语学习方式，但在备考学术英语的过程中，花上一个晚上的时间段竟然完全可以一口气记下三个单元的单词。

@residualsun1  学语言免不了背单词，这是比较无聊的，最好能趁早习惯。词汇量是根基，词汇量低了，听说读写统统有障碍。我看你待办里写了默写单词，我推荐 Anki。自己默写我总是不知道该什么时候复习，Anki 会给你自动安排复习，记得好的单词更迟出现，记得差的更快出现。Anki 牌组还可以分享给别人，大学那会我就挺喜欢分享给同学。

@liechi 背单词我可以，但背文章好难，上学那会觉得这个特别煎熬。我的语感是通过阅读真实的英语材料（书、文章、网络帖子等）建立的。葛传椝就说多读英语写的书（books in English），少读论述英语的书（books on English，例如语法书）。

**Reply by residualsun1** [reply]
> 背诵做到可以不想就能把内容脱口而出就好了，这是建立语感的途径之一。

> 我的语感是通过阅读真实的英语材料（书、文章、网络帖子等）建立的。

我高中的英语就是通过纯粹的阅读理解来提高的。说起来很是怀念高一的暑假，那时候学校要求我们提前一个月返校自习。我每天晚上就固定做一件事：做2篇阅读理解和1篇填空题，做完慢慢对答案，然后自己重新把题目翻译一遍。一直到开学，我后来才发现英语基础打下来了，平时即使不怎么背单词，但是靠阅读理解积累的语感好像也够用了，只要平时保持一定的阅读量就好。

> 词汇量是根基，词汇量低了，听说读写统统有障碍。我看你待办里写了默写单词，我推荐 Anki。

不过现在可能要改观了，这种方法的缺陷是我对单词的理解停留在联想上，很多时候拼写会出现错误。所以也还是要回到背单词。哈哈，感觉叶寻有好多干货，收到安利啦。上次推荐的Joplin也很好用！不过我好像更习惯直接在Rstudio里写（比较零碎的内容），之后认真写长文的时候会用上Joplin。

> 按我自己的经验，学外语最有效的方法就是背诵和重复了，包括被单词和文章。

另，列驰说的背文章我也觉得有点难度。不过我喜欢一些不是很长的英语演讲。有个up主叫向杨，我喜欢他的“爵士伦理学”栏目，里面主要是Cornel West 哲学教授的谈话。我从[《勇敢是我们最稀缺的品格，因为冷漠最容易传染》](https://www.bilibili.com/video/BV1Ma411G7BZ/?spm_id_from=333.999.0.0&vd_source=8fedd318b0a64ca095a2dae4efeeda6e)开始入坑，发现无论是演讲的内容还是腔调都有一种能让我背下去的冲动。

**Reply by residualsun1** [reply]
> 这是一个组还是三个组？万物有灵，听起来很有意思，跟修仙似的。另外两个也很玄的样子。

是一个兴趣小组，目前是大约有一个大方向，但是还没正式形成最终的6人小组。

我也觉得很有意思，虽然也不懂，但也入了兴趣组，观望一下。其实很多部族给我的感受都是相信万物有灵的。“万物有灵、非人、物和本体论”是人类学现在比较关注的领域，我也是刚刚入门，只有一些不成熟的感性认识。我准备看的《森林如何思考》，这本书是关于厄瓜多拉的阿维拉民族志，故事源于作者准备安营扎寨休息时，得到的当地人的一段警告：

> “要仰面朝上睡觉！如果美洲豹来了，看到你可以回头看他，他就不会打扰你。如果你面朝地下睡觉，他就会认为你是‘aicha’，那他就会发动攻击。”

这句话启发作者尝试作为一只美洲豹来理解自然、社会以及人类本身，并且延伸出森林是否会思考、狗是否会做梦、树和猴子间的交流方法等等话题。我想这可能是万物有灵的一个体现。没有疑问的是这体现了去人类中心化（从人以外的物思考世界）和本体论（“人”自身是什么），正如文中提到当地人相信存在一种鲁纳美洲豹人，可以从人变成美洲豹。

端午节和兵哥吃饭，我才知道《末日松茸》也是人类学本体论的方向。我的粗浅理解是，整本书可以说是以松茸以及与松茸发生交染的松树、橡树以及森林为主角，作者有意分开人类的资本主义、规模化、追求确定性的视角和属于以松茸为代表的历史性、多样性和不确定性的视角，又将两个叙述角度以带有诗意的散文般的形式“缠绕”在一起，给我的感觉像是要把我们从人类的视角轻轻拉入松茸和森林的视角。这样想或许也是万物有灵的一种表现吧！

最后，我没想到会让你觉得有修仙的感觉哈哈哈，但玄乎是有一点玄乎的！我就一直好奇关于原始部落的萨满祭司，还有部族人们关于森林、万物的信仰等等。或许这也是人类学的魅力吧，在科学、技术、历史、文化等角度之外，我们还有别的方式来认识自身，以及与自身以外他者的关系。

**Reply by liechi** [reply]
原来是这么一个“有灵”法啊，那跟修仙没多大关系了。有个哲学家叫伯格森，他提倡进入对象去思考，和它们共感，共历，而不要从外部去“研究”它们。很玄的理论，也不知道他“进入”对象后看到了什么。不过这个道理听起来倒是跟你说的万物有灵有些联系。

《幽灵公主》里，人们把森林当做开发利用的对象，而幽灵公主把森林本身以及里面生活的动物们当做朋友，为保护它们而带头反抗人类。这算是一个体现万物有灵和科技指引下社会发展之间矛盾的有趣例子。

或许，在现代社会里，被异化为物而丧失“灵”的不仅是蜂蝶草树，还包括人本身。我觉得现代社会也不是人类中心化的，而是去人类中心，只是比起其他物件来，人离那个“中心”稍微近些。那个“中心”是什么？人努力地活着，努力工作，一关一关地过，目的是什么？意义是什么？这好像又绕到你之前写的一篇博客的主题上来了。我平时不太想这些，至少是最近些年来不怎么想了。如果人本身不是目的和意义，那这些问题就太麻烦了，想不清楚；如果人本身是目的和意义（人的灵是什么），那怎么把它活出来？

**Reply by CyrusYip** [reply]
> 我后来才发现英语基础打下来了，平时即使不怎么背单词，但是靠阅读理解积累的语感好像也够用了，只要平时保持一定的阅读量就好。

@residualsun1 名师旋元佑就是读书不求甚解，不查单词的。不过他阅读量很大，可以一晚上看一本小说。我备考英语专业四级的时候发现自己词汇量不足，但又不想死背词汇书，就去读小说积累词汇了。我用的阅读方法是透析法，出自《把你的英语用起来！》。那会应该涨了2000词汇量，练了听力，做了十套试卷，最后顺利通过。读小说可比被词汇书好玩多了。

阅读读得多了你就知道一些单词的细微差距。

> Linus Benedict Torvalds is a Finnish software engineer who is the creator and, historically, the lead developer of the Linux kernel

以程序员为例，以编程为职业是 software engineer，开发某个软件是 developer。背单词要在句子段落中被背，不能单独抽出来死记。

> 这种方法的缺陷是我对单词的理解停留在联想上，很多时候拼写会出现错误。

你学过自然拼读法吗？我上学的时候好像老师教过。根据发音背写法会比较容易记。不过有些词是不符合自然拼读法的，例如出自法语的 reservoir（水库）。

**Reply by residualsun1** [reply]
> 原来是这么一个“有灵”法啊，那跟修仙没多大关系了。

@liechi 我也是初次了解，是感性的经验回答哈哈，和专业远远拉不上关系。希望这一次去听课能有所收获，回来有机会的话再补充(●‘◡’●)。

> 也不知道他“进入”对象后看到了什么。

这也是我的疑惑，不知道那些人类学家是怎么做到能让脑子冒出一些“惊人”的想法。唔，不过实践出真知，人类学家的田野调查至少要有一年及以上，咱没干过这活，不理解也很正常。越是这样反而越是好奇“进入”之后看到的是啥。

> 只是比起其他物件来，人离那个“中心”稍微近些。那个“中心”是什么？人努力地活着，努力工作，一关一关地过，目的是什么？意义是什么？

关于“中心”的话，想起项飙在《把自己作为方法》一书中有过相关讨论，不过是围绕“中心”和“边缘”的关系，大概是讲现在年轻人都喜欢往“中心”去挤，但是忽略了“边缘“，可以尝试去发现”边缘“的特色、特点。我把这里的”中心“理解成体面的、代表更高地位的工作、以高楼大厦、各式先进交通设施等为代表的社会景观，”边缘“则更像是自己的家乡，比如家乡有啥特色，其实也是有某些机会的（好像项飙也是以从地理性的角度出发讨论）。我家里人都很喜欢家乡的环境，比起在大城市生活，他们更喜欢虽然收入一般、也没有那么富丽繁华但是慢悠悠的县城生活。我观察到，在我上大学后，我父母周末时不时会约上三五好友爬山游泳；我有一个舅舅经营着果园，平时打理完果园就喜欢开着车从县城到不同的村子闲逛，看不同的乡村的风景，用他的话就是喜欢这种田园风光，这就是他的生活。当时看完项飙关于“中心”和“边缘”关系的讨论，我就想到了我父母那一辈，至少我家的长辈对于“中心”好像就没有很强的向往。想起这些之后，想法也随着键盘输出来啦。我想，不同的阶段每个人的“中心”具体都不大一样，可能是眼前一地鸡毛和奋力前进的现实，也可能是关于个人存在与意义的形而上追问。我这时候会更加倾向想到工作和生活。但我想答案或许不重要，重要的是我们偶尔会想起它，思考它，并书写它（无论是啥方式）（￣︶￣）。

> 如果人本身不是目的和意义，那这些问题就太麻烦了，想不清楚；如果人本身是目的和意义（人的灵是什么），那怎么把它活出来？

我觉得想不清楚是一件好事。小时候我会想宇宙有多大，虽然科学和课本的答案是无限大，但我会接着想：”‘**无限大’有多大？**”然后想象着一艘飞船飞了很久很久，最后抵达宇宙的边界，就好像抵达鸡蛋壳的某个边缘，接着继续猜想边界破碎之后的外界又是什么样子……最后的结果就是把自己想懵了。我想，有时候我们觉得想不清楚，是因为外界的答案和个人的答案之间的平衡和取舍。外界无法给我们提供一个完全信服的答案，那都是别人的经验，不管是哪个生物学家还是人类学家，他们说的只能是对一部分的参考。但我们也无法做到坚定地相信自己的选择，也总是犹犹豫豫，磕磕绊绊。同时外界和自我又交织起来，我感觉是无法得到一个确定的答案，那么这些问题只能一直去探索，这个过程又给我们留下了多样性和不确定性，允许缠绕，允许所谓正义邪恶，允许悲欢离合，允许每个人给生命交上一份无标准答案的考卷。所以，我想或许“有意识地”活着从某种程度上来说就是活出自己的目的、意识和灵。

**Reply by residualsun1** [reply]
> 我备考英语专业四级的时候发现自己词汇量不足，但又不想死背词汇书，就去读小说积累词汇了。我用的阅读方法是透析法，出自《把你的英语用起来！》。那会应该涨了2000词汇量，练了听力，做了十套试卷，最后顺利通过。读小说可比被词汇书好玩多了。

@CyrusYip 嘿嘿，其实翻你博客的时候看到你给学计算机的访客推荐过，我就上z-library下了电子版。本来寄希望于读英文文献，但事实证明我太狂妄了，中文文献都读不进去(T_T)，我竟然还指望英文文献。因为还在学R，我想我可以先从bookdown文档里面的电子书（里头的英文读起来也挺容易接受的）开始。不过还是觉得你们去读小说真是狠人(ﾟДﾟ)……

> 以程序员为例，以编程为职业是 software engineer，开发某个软件是 developer。背单词要在句子段落中被背，不能单独抽出来死记。

虽然我没有花上时间系统地实践过，但有过去的阅读经验，我双手双脚赞成“不能单独抽出来死记”。就以你举的例子来说，我觉得这种学习（深挖，拆开，嚼碎）体验不仅是利于长远的持续学习，也是发现乐趣的一个途径。

> 你学过自然拼读法吗？我上学的时候好像老师教过。根据发音背写法会比较容易记。不过有些词是不符合自然拼读法的，例如出自法语的 reservoir（水库）。

听说过的，平时写英语单词不自觉也会在心里念叨，然后凭感觉写出个大概，但是没有怎么学过。我出错的情况大多是单词形象熟悉度很低，或者是明显和读音有出入（也可能是我自己并不真正了解自然拼读法）。不过我觉得如果音标熟悉了，能比较准确地念出单词，然后多结合单词的形象，熟能生巧，效果就会好。

**Reply by CyrusYip** [reply]
> 不过还是觉得你们去读小说真是狠人(ﾟДﾟ)……

@residualsun1 读小说学单词比读文章容易多了，是不是很反直觉？因为小说更长，有固定的主题，里面的单词会重复出现，背过的一直看就相当于一直复习，那些不懂的单词看多了也许能猜到大概意思。

小说也有难易之分，The Boxcar Children 这种写给小学生的小说就比较简单，我备考的时候读了十多本。

> 不过我觉得如果音标熟悉了，能比较准确地念出单词，然后多结合单词的形象，熟能生巧，效果就会好。

音标是必学的，不然读不准。你说的单词形象是啥意思？

> 嘿嘿，其实翻你博客的时候看到你给学计算机的访客推荐过，我就上z-library下了电子版。

分享一下我看纸质书的省钱窍门：多抓鱼（二手）、图书馆（免费）。

**Reply by residualsun1** [reply]
> 因为小说更长，有固定的主题，里面的单词会重复出现，背过的一直看就相当于一直复习，那些不懂的单词看多了也许能猜到大概意思。

@CyrusYip 嗯，我觉得这个很有说服力。

> 你说的单词形象是啥意思？

就是单词的样子，比如说一个单词比较长，例如**psychotherapist**（心理医生），我要大致知道它的“容貌”中有cho、the、rap、ist这几个要素，然后就是它们的排列顺序。不过如果能念准音标一般能大致写出来，但是我可能会漏掉某个字母，这个时候如果脑子里大致有这个要素的印象，结合音标读音就可以比较准确地写出来。

> 分享一下我看纸质书的省钱窍门：多抓鱼（二手）、图书馆（免费）

第一个途径性价比感觉挺高的。我喜欢在书上涂涂画画，所以图书馆的书借得比较少。
